### DISCUSSION TITLE
Very slow generation speed

### DISCUSSION THREAD

**Discussion by budeni** [oobabooga/text-generation-webui#1239] [General]
Hi i have 3090 24gb but it takes around 5-10 second (1.65 tokens/s) to respond simple questions like "hi" Is there a way to fix that?

**Comment by dexterhuh** [comment]
I got a 3060 mobile version. got like 20 seconds for Hi. I was wondering too if there is a way can speed up the response. I got a desktop AMD 3090 will that be faster if I use the CPU version?
